src/gcp/cloudfunctions.ts

```diff
@@ -383,9 +381,7 @@ export async function updateFunction(
   // a protobuf.Empty value, so we have to manually add it in the missing case.
   try {
     const headers: Record<string, string> = {};
-    if (previews.artifactregistry) {
```

Comment:
It looks like we can get rid of this preview flag altogther in `previews.ts`

Reply:
Yup. Looks like I accidentally forgot to save that file.